make validation privacy optional (#28)
OR13/diem-draft-charter: diem-unified-charter.md

A discovery mechanism for the initial work will only be specified by this group after the initial emblem binding protocol is completed (see the Deliverables section below).

The working group is initially limited to specifying discovery mechanisms that rely on digital communication such as the use of DNS or well-known locations on a host identified by a hostname or IP address.

> Only discovery mechanisms where the validation, by default, remains unknown to the bearer of the emblem are considered in the initial scope.

Reviewer: ```suggestion
Only discovery and binding mechanisms where the validation, by default, remains undetectable as such are considered in the initial scope.
```

Hi Rohan, just wondering: Would you be opposed to this change? I think it phrases the underlying requirement more accurately.

Author: I think the other wording (originally from Mirje) is clearer.
